### DISCUSSION TITLE
Issues with pyjq 2.6.0 when installing on Windows

### DISCUSSION THREAD

**Discussion by theagilepilot** [port-labs/ocean#837] [Q&A]
I've been getting issues attempting to install ocean framework with a regular pip install and also with poetry.

The error always results when the install reachespyjq. I get the following error with poetry:

error: [WinError 2] The system cannot find the file specified

  at ~\pipx\venvs\poetry\Lib\site-packages\poetry\installation\chef.py:164 in _prepare
      160│
      161│                 error = ChefBuildError("\n\n".join(message_parts))
      162│
      163│             if error is not None:
    → 164│                 raise error from None
      165│
      166│             return path
      167│
      168│     def _prepare_sdist(self, archive: Path, destination: Path | None = None) -> Path:

Note: This error originates from the build backend, and is likely not a problem with poetry but with pyjq (2.6.0) not supporting PEP 517 builds. You can verify this by running 'pip wheel --no-cache-dir --use-pep517 "pyjq (==2.6.0)"'.

Does anyone have a solution? (Im using python 3.11.9)

**Comment by PeyGis** [comment]
Hello @nicholasamato thanks for reporting this and sorry for the very late reply. Users who have faced similar issues in the past were able to overcome this challenge by following these steps:

1. Update the poetry to create/use a virtual env based on 3.11 `poetry env use python3.11`
2. Then activated the env `. .venv/bin/activate`

Please let me know if this resolves your issue
